doc/source/howtos/upgrade_guide.rst

```diff
@@ -117,3 +120,41 @@ To upgrade the cluster, follow these steps for each node:
 #. Repeat this procedure for the next node in the cluster
    until you upgrade all nodes.
 
+It is important that when rejoining, the node should synchronize using
+:term:`IST`.
```

Comment:
s/./. For this, it is best not to leave the cluster node being upgraded offline for an extended period. More on this below./